dist/main.js

```diff
@@ -0,0 +1,10 @@
+$("#find-roster").on("click", function(){
+    const teamName = $("input").val()
+    console.log(teamName)
+    $.get(`teams/${teamName}`, function(roster){
+        const source = $("#content").html()
+        const template = Handlebars.compile(source)
+        const teamContent = template({ teamRelevantData : roster })
+        $(".main-content").append(teamContent)
```

Comment:
Looks like you are missing the `empty()` command. I don't think this will work a second time.